app/src/full/java/io/homeassistant/companion/android/sensors/HealthConnectSensorManager.kt

```diff
@@ -190,6 +228,18 @@ class HealthConnectSensorManager : SensorManager {
             entityCategory = SensorManager.ENTITY_CATEGORY_DIAGNOSTIC
         )
 
+        val hydration = SensorManager.BasicSensor(
+            id = "health_connect_hydration",
+            type = "sensor",
+            commonR.string.basic_sensor_name_hydration,
+            commonR.string.sensor_description_hydration,
+            "mdi:cup-water",
+            unitOfMeasurement = "mL",
```

Comment:
This can be `ml`, no need to capitalize the liter part of it

Reply:
gemini autocomplete got the best of me there 🙈